Create osu! Beatmapping World Championship Newspost (#14623)
ppy/osu-wiki: news/2026/2026-04-18-osu-beatmapping-world-championship-2026.md

---
layout: post
title: osu! Beatmapping World Championship 2026
date: 2026-04-18 20:00:00 +0000
series: community_contests
---

The **osu! Beatmapping World Championship (o!bwc)** returns for 2026!

![](/wiki/shared/news/2026-04-18-osu-beatmapping-world-championship-2026/banner.jpg)

[o!bwc](/wiki/Contests/o!bwc) is an international beatmapping contest that brings together beatmappers from around the world to showcase their creativity, expertise, and passion for mapping. Teams represent their home country and compete through qualifiers and intense clash rounds, all in pursuit of the championship title.

> This year’s edition is hosted by ::{ flag=KR }:: [momoyo](https://osu.ppy.sh/users/12469536), ::{ flag=US }:: [-White](https://osu.ppy.sh/users/16276548), ::{ flag=HK }:: [Chaoslitz](https://osu.ppy.sh/users/3621552), and ::{ flag=CN }:: [Mafumafu](https://osu.ppy.sh/users/3076909). Whether you’re a seasoned mapper or just getting started, gather your team and take part! Join the [Discord server](https://discord.com/invite/QGzz3n655u), visit the [forum thread](https://osu.ppy.sh/community/forums/topics/2196621), or register directly [here](https://form-auth.ppy.sh/form/2026-bwc-registrations).

Reviewer: https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#serial-comma

```suggestion
This year's edition is hosted by ::{ flag=KR }:: [momoyo](https://osu.ppy.sh/users/12469536), ::{ flag=US }:: [-White](https://osu.ppy.sh/users/16276548), ::{ flag=HK }:: [Chaoslitz](https://osu.ppy.sh/users/3621552) and ::{ flag=CN }:: [Mafumafu](https://osu.ppy.sh/users/3076909). Whether you're a seasoned mapper or just getting started, gather your team and take part! Join the [Discord server](https://discord.com/invite/QGzz3n655u), visit the [forum thread](https://osu.ppy.sh/community/forums/topics/2196621) or register directly [here](https://form-auth.ppy.sh/form/2026-bwc-registrations).
```

## Schedule

| Event | Dates |
| --: | :-- |
| Registration | April 18 – May 2 |
| Screening | May 2 – May 9 |
| **Qualifier Round** | May 9 – June 6 |
| **Round of 16** | June 6 – July 4 |
| **Quarterfinals** | July 4 – July 25 |
| **Semifinals** | July 25 – August 15 |
| **Finals** | August 15 – September 5 |

The schedule may change slightly depending on the progress of the contest.

## Format

**o!bwc** is a team-based beatmapping contest for the **osu! game mode**, beginning with a ranking-based qualifier round before moving into single-elimination clash rounds.

At the start of each stage, teams are given a new set of songs to map. Once the competition reaches the clash stage, teams face off in head-to-head matchups, where a ban-pick phase at the beginning of each round adds an extra strategic element alongside the mapping itself.

## Registration

**[Click here to register for this contest!](https://form-auth.ppy.sh/form/2026-bwc-registrations)**

> Teams consist of 4–6 members representing the same country. If multiple teams register under the same flag, only the highest-ranking team from qualifiers will advance to the knockout stage. Team captains must join the contest’s [Discord server](https://discord.com/invite/QGzz3n655u) to stay up to date with announcements. Other members in the team are also strongly encouraged to join.

Reviewer: ```suggestion
Teams consist of 4–6 members representing the same country. If multiple teams register under the same flag, only the highest-ranking team from the qualifiers will advance to the knockout stage. Team captains must join the contest's [Discord server](https://discord.com/invite/QGzz3n655u) to stay up to date with announcements. Other members in the team are also strongly encouraged to join.
```

## Beatmapping and submission

> Each team submits a single map per round, along with a clear list of contributions from members. Submissions should follow the [ranking criteria](/wiki/Ranking_criteria) where applicable and must remain anonymous and unpublished until results each round are revealed. Custom hitsounds and combo colours are allowed, but backgrounds, videos, and storyboards are not permitted. Metadata such as artist and tags should not be modified either.

Reviewer: ```suggestion
Each team submits a single map per round, along with a clear list of contributions from members. Submissions should follow the [ranking criteria](/wiki/Ranking_criteria) where applicable and must remain anonymous and unpublished until results each round are revealed. Custom hitsounds and combo colours are allowed, but backgrounds, videos and storyboards are not permitted. Metadata such as artist and tags should not be modified either.
```

## Judging

> There will be two judging panels: one for the qualifier round and one for the clash rounds. In qualifiers, submissions are evaluated based on **expertise**, **creativity**, and overall **impression**, with scores standardised across judges. In clash rounds, judging shifts to a voting system where each matchup is decided simply by majority vote, with the winning team advances to the next.

Reviewer: ```suggestion
There will be two judging panels: one for the qualifier round and one for the clash rounds. In the qualifiers, submissions are evaluated based on **expertise**, **creativity**, and overall **impression**, with scores standardised across judges. In clash rounds, judging shifts to a voting system where each matchup is decided simply by majority vote, with the winning team advancing to the next round.
```

> ## Prizes and Awards

Reviewer: ```suggestion
## Prizes and awards
```

> Top teams may receive Elite Mapper Points, profile badges, and osu!supporter or cash prizes after the contest concludes, depending on sponsorship and approval.

Reviewer: ```suggestion
Top teams may receive [contest points](/wiki/Contests/Contest_points), profile badges, and osu!supporter or cash prizes after the contest concludes, depending on sponsorship and approval.
```
